src/endpointDefinitions.ts

```diff
@@ -76,9 +76,18 @@ type ResultDescription<EntityTypes extends string, ResultType, QueryArg, ErrorTy
   | GetResultDescriptionFn<EntityTypes, ResultType, QueryArg, ErrorType>;
 
 export interface QueryApi<ReducerPath extends string, Context extends {}> {
+  /**
+   * The dispatch method for the store
+   */
   dispatch: ThunkDispatch<any, any, AnyAction>;
+  /**
+   * A method to get the current state
+   */
   getState(): RootState<any, any, ReducerPath>;
   extra: unknown;
+  /**
+   * A unique ID generated for the mutation
+   */
   requestId: string;
   context: Context;
```

Comment:
```suggestion
  /**
  * A variable shared between `onStart`, `onError` and `onSuccess` of one request to pass data forward between them
  */
  context: Context;
```

Reply:
Thanks, have you got a description that would be suitable for `extra` too? I left that blank for now